### ISSUE TITLE
Questions regarding to dataloader

### ISSUE THREAD

**Issue by crisb-DUT** [victoresque/pytorch-template#6]
The project is very exciting, but I don't understand why we should implement some functions in 
 dataloader.py, such as shuffle, batch_szie and so on, which have already been implemented in torch.utils.data.DataLoader. 
If we could make the dataloader lighter?

Thanks.

**Comment by victoresque** [comment]
Hi, thanks for the comment!

Yes, I'll deprecate `BaseDataLoader` later. But for now, there's no dependency between `BaseDataLoader` and other parts, so one can simply change the data loader in `train.py` to using `torch.utils.data.DataLoader` and `torch.utils.data.Dataset`.

In conclusion using `torch.utils.data` could be a better option, thanks!

Victor
